.env.development

```diff
@@ -4,6 +4,10 @@ BASE_URL='localhost:1994'
 LMS_BASE_URL='http://localhost:18000'
 LOGIN_URL='http://localhost:18000/login'
 LOGOUT_URL='http://localhost:18000/login'
+LOGO_URL=https://edx-cdn.org/v3/default/logo.svg
```

Comment:
do these have to be updated in other envs too?

Reply:
Not according to the Flip Instructions doc.  Just .env.development (and .env.test if applicable, but we don't have that one)
https://openedx.atlassian.net/wiki/spaces/AC/pages/2047345474/Logo+Flip+Plan+and+Instructions#MFE-Logo-Instructions

Reply:
neat-o. I'm definitely gonna check out that doc